maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -412,11 +441,14 @@ data class ApplyConfigurationCommand(
 data class OpenLinkCommand(
     val link: String,
     val autoVerify: Boolean? = null,
-    val browser: Boolean? = null
+    val browser: Boolean? = null,
+    val label: String? = null,
 ) : Command {
 
     override fun description(): String {
-        return if (browser == true) {
+        return if (label != null && label != "") {
```

Comment:
```suggestion
        return if (label != null) {
```

See my comment on `Condition.kt`